neuralprophet/uncertainty.py

```diff
@@ -77,39 +78,21 @@ def predict(self, df: pd.DataFrame, df_cal: pd.DataFrame) -> pd.DataFrame:
             noncon_scores = self._get_nonconformity_scores(df_cal, step_number)
             q_hat = self._get_q_hat(df_cal, noncon_scores)
             y_hat_col = f"yhat{step_number}"
+            y_hat_lo_col = f"{y_hat_col} {min(self.quantiles) * 100}%"
+            y_hat_hi_col = f"{y_hat_col} {max(self.quantiles) * 100}%"
             if self.method == "naive" and self.symmetrical:
                 q_hat_sym = q_hat["q_hat_sym"]
-                q_hat_col = f"qhat{step_number}"
-                df[q_hat_col] = q_hat_sym
-                df[f"{y_hat_col} - {q_hat_col}"] = df[y_hat_col] - q_hat_sym
-                df[f"{y_hat_col} + {q_hat_col}"] = df[y_hat_col] + q_hat_sym
+                df[y_hat_lo_col] = df[y_hat_col] - q_hat_sym
+                df[y_hat_hi_col] = df[y_hat_col] + q_hat_sym
             elif self.method == "cqr" and self.symmetrical:
                 q_hat_sym = q_hat["q_hat_sym"]
-                q_hat_col = f"qhat{step_number}"
-                df[q_hat_col] = q_hat_sym
```

Comment:
Idea is that for the advanced users, they need to input the `q_hat` into the forecast df themselves using the noncon scores right?